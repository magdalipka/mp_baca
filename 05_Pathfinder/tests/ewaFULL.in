5 8
1 0 1 0 1
1 1 0 0 0
1 1 0 1 0
0 0 0 1 1
1 1 0 1 1
1 0 0 0 1
0 0 1 0 0
0 1 1 1 0
12
r 1 7 3 7
i 1 7 3 7
r 3 7 4 5
i 3 7 4 5
r 0 0 4 5
i 0 0 4 5
r 3 1 0 0
i 3 1 0 0
r 2 4 4 5
i 2 4 4 5
r 0 0 0 0
i 0 0 0 0

