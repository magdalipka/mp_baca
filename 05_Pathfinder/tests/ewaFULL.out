r X
i X
r S E S
i S E S
r N E N E N N N N E E S
i N E N E N N N N E E S
r N W W S W S
i N W W S W S
r N N E E S
i N N E E S
r
i
